metricbeat/module/couchbase/bucket/bucket_integration_test.go

```diff
@@ -22,15 +22,37 @@ package bucket
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
+	"github.com/elastic/beats/libbeat/tests/compose"
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 	"github.com/elastic/beats/metricbeat/module/couchbase"
 )
 
+func TestFetch(t *testing.T) {
+	compose.EnsureUp(t, "couchbase")
+
+	f := mbtest.NewReportingMetricSetV2(t, getConfig())
+	events, errs := mbtest.ReportingFetchV2(f)
+	if len(errs) > 0 {
+		t.Fatalf("Expected 0 error, had %d. %v\n", len(errs), errs)
+	}
+
+	assert.NotEmpty(t, events)
+	t.Logf("%s/%s event: %+v", f.Module().Name(), f.Name(), events[0])
+}
+
 func TestData(t *testing.T) {
```

Comment:
As commented in other PR's, to avoid potential flaky tests `TestData` must now look like this:

```go
func TestData(t *testing.T) {
	f := mbtest.NewReportingMetricSetV2Error(t, getConfig())

	if err := mbtest.WriteEventsReporterV2Error(f, t, ""); err != nil {
		t.Fatal("write", err)
	}
}
```